code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -541,13 +541,11 @@
 		if(!emagged)
 			to_chat(user, "<span class='warning'>The cart is working fine and doesn't needs repairs!</span>")
 			return
-		if(!user.drop_item())
-			to_chat(user, "<span class='warning'>\The [O] is stuck to you!</span>")
+		if(!user.unEquip())
+			to_chat(user, "<span class='warning'>[O] is stuck to you!</span>")
 			return FALSE
-		user.drop_item()
 		new_electronics.forceMove(src)
-		spawn_atom_to_turf(/obj/item/airlock_electronics/destroyed, user.loc, 1)
-		emagged = FALSE
```

Comment:
did you mean to delete this line?

Reply:
@SchrodingersWolf ???

Reply:
Sorry - Got really demotivated. Yeah, that was not supposed to be removed, it is back in now.

Thanks!